### ISSUE TITLE
Deploy ML model using Flask

### ISSUE THREAD

**Issue by Roshan23R** [Kali-Decoder/Eco_GRiD_H23#27]

**Comment by ashishlal2003** [comment]
Would there be a single "/predict" route based on the input values from columns decided? Also, how would the folder structure look like in the "ml-part" directory?

**Comment by Roshan23R** [comment]
@ashishlal2003 Yes for now you can simply keep "/predict" route based on important columns you have decided for prediction. Keep in the same directory.
